### DISCUSSION TITLE
Notifications

### DISCUSSION THREAD

**Discussion by fordcrews** [Blackymas/NSPanel_HA_Blueprint#1015] [General]
Are the notifications shown here for a future version?    I can't seem to find how/where to configure any of these.  [https://user-images.githubusercontent.com/41958506/212768891-9610d800-43c5-454f-a8b6-6b7c43677b5d.png](url)
![212768891-9610d800-43c5-454f-a8b6-6b7c43677b5d](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/17262935/25efaa64-e7f9-448e-923d-d8a963366f43)

**Comment by Doughboy68** [comment]
I cannot see that page, but for as long as I remember, were always able to send notifications to the panel via a service in HA.
ESPHome: {nspanelname}_notification_show

**Reply by edwardtfn** [reply]
That's right. Notification icon will be shown when a notification is created with a service call to `esphome.xxxxx_notification_show`.

**Reply by fordcrews** [reply]
So I make a helper script, that if a door is open it calls  

service: esphome.xxxx_notification_show
data:
  label: Door Open
  text: Front Door Open

for all the panels I have?  The image shows icons, not labels/text.   

I just thought it would be cool to have a garbage can on garbage day, a door open icon if a window or door is open, a fire icon if my computer closet is over hot, a high humidity if the humidity is high, etc on all my panels.  ;-)   I tend to get carried away.

**Reply by edwardtfn** [reply]
If you don't click check at notification page (you can click the close button or wait for page timeout) it will go back to home page and now a notification icon (mdi:email) is visible. Click there to see the notification again.

**Reply by edwardtfn** [reply]
> I just thought it would be cool to have a garbage can on garbage day, a door open icon if a window or door is open, a fire icon if my computer closet is over hot, a high humidity if the humidity is high, etc on all my panels. ;-) I tend to get carried away.

You can use the chips (icons in the top of the screen) for that. When the assigned entity is on/enable/true, the icon will be visible, otherwise it will be hidden.

**Reply by fordcrews** [reply]
I use the chips currently for motion events, watch them flicker on and off as people move around.     I just saw the notification icons in the sample image and could not find any documentation on them.
